setup.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+ADAFRUIT_DIR=www/Adafruit_Python_GPIO
+
+if [ "${EUID}" -ne 0 ]; then
+	echo "Error: This script must be run as root."
+	exit 1
+fi
+
+if [ ! -e ${ADAFRUIT_DIR} ]; then
+	echo "Error: Adafruit directory not found."
+	exit 2
+fi
+
+apt-get install -y python python-dev python-pip python-smbus lighttpd
```

Comment:
If we're going to install lighttpd for them, maybe we go as far as to provide them with the lighttpd.conf that they'll need (with the mod_proxy set to foward to our port 5000 Flask app)?
We could either edit the default lighttpd.conf with this script, or keep ours in git and copy it over their default one.
Thoughts?